contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -945,22 +926,7 @@ class tsqlCommonMutator : public TSqlParserBaseListener
 	 */
 	void exitLocal_id_expr(TSqlParser::Local_id_exprContext *ctx) override
```

Comment:
Is there a better way NOT to use local id? Its the lowest level and its not optimal at all

Reply:
So basically it's not local_id but it's Local_id_expr
```
expression    
       : local_id (DOT calls+=method_call)*                                        #local_id_expr
``` 
Here method_call can be spatial_methods

and yeah we cannot ignore this part as it is one important category of expression which needs to be handled for CLR_UDT methods for example:
1. @point.STX
2. @point.STAsText()
3. @point1.StDistance(@pnt)

These all expressions will fall under this category and need to be handled separetely, and it's not the lowest level as we already have handlings for same level expressions such as ```full_column_name```

Reply:
Why do we have to do it this way? Can we add a single rule directly into expression ?

Reply:
Distinguish with method call keywords